src/main/java/org/gridsuite/study/server/InsensitiveStringToEnumConverterFactory.java

```diff
@@ -0,0 +1,85 @@
+/*
```

Comment:
that because I copied this file from Spring codebase and keep delete parts I don't need (no major code added). Did I replace the licence of the file in this case?

Reply:
I don't know, you should ask a TL maybe

Reply:
TL say to replace for short code like this